case-app/angular-library
# feat(dropdownLinks): add disabled and tooltip properties

## What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)

This PR is adding two new properties to dropdown link 

## Description

Adding disabled and tooltip properties to dropdown link.

Disabled is a function expecting a boolean if it's true, the dropdown item will be disabled (seen but not clickable) 
 Tooltip is a string which appears on hover


## How can it be tested?

run a CASE project linked with this angular library. 
Add these properties to a dropdown link object. 

You can check the PR on the CASE Docs to see an example of how used them.
